packages/trpc/server/routers/viewer/eventTypes/update.handler.ts

```diff
@@ -225,12 +226,10 @@ export const updateHandler = async ({ ctx, input }: UpdateOptions) => {
   }
 
   if (teamId && hosts) {
-    // check if all hosts can be assigned (memberships that have accepted invite)
     const memberships =
       (await ctx.prisma.membership.findMany({
         where: {
           teamId,
-          accepted: true,
```

Comment:
Why was this removed?

Reply:
https://github.com/calcom/cal.com/blob/bef1bf6afeedd270c8e8b154af5621ed3abb7f63/packages/trpc/server/routers/viewer/eventTypes/update.handler.ts#L235-L243

Earlier pending members couldn't be assigned as hosts so we only needed to check for accepted ones, this isn't the case now